addon/components/frost-fixed-table.js

```diff
@@ -22,14 +22,17 @@ export default Component.extend({
   propTypes: {
     // options
     columns: PropTypes.arrayOf(ColumnPropType),
-    items: ItemsPropType
+    items: ItemsPropType,
```

Comment:
Yeah, makes sense for them to be required, `items` can be required for `frost-table` too, since it's not much of a table without a list of things, even an empty list.